requirements/prod.txt

```diff
@@ -18,3 +18,5 @@ requests
 
 # Metrics collection
 python-statsd==2.1.0
+
+git+https://github.com/trustbridge/libtrustbridge@6d607bd6#egg=libtrustbridge
```

Comment:
in the future it might be branch or tag instead of just commit; because tag may be re-pushed and branch may get security or bug fixes; and this approach still allow us to freeze the code API version to specific state.

Reply:
yep, tagging is more appropriate, I just wanted to keep it simple until we decide our release strategy.

Reply:
pypi releases could make sense too.

Reply:
A bit more painful whilst doing development though... And TBH, I don't love the idea of loading pypi up with packages that are only for a specific project (I'm sure there are plenty of them anyway, though).

Reply:
yes, even if we use PyPi let's not do it during the active development phase (until we can provide more or less stable API)